### PR TITLE
Kasli: Added front panel user LED (#1623)

### LINKED ISSUE #1623: connect Kasli 2.0 front panel USER LED to RTIO
The ARTIQ core device tutorial calls for using a RTIO LED, which is causing confusion to new Kasli 2.0 users as there is no RTIO LED by default. It is straightforward for more experienced people to use a TTL instead, but this isn't too friendly to beginners.

So a front panel USER LED should be connected to RTIO and present in the DDB templates.

Similarly, I think the board USER LED should not be connected to RTIO so that it cannot be inadvertently used by people taking the tutorial for the first time with a Kasli card already in a crate where the board LED isn't visible.

### PR DESCRIPTION
**Pull request by SidaChen1999**
# ARTIQ Pull Request

## Description of Changes
Fix the problem described by issue #1623. Now we can control 2 LEDs on the front panel of Kasli v2.0 like the documentation of ARTIQ

### Related Issue
Closes #1623 

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :sparkles: New feature |

### All Pull Requests

- [x] Close/update issues.

### Code Changes

- [x] Test your changes or have someone test them. Mention what was tested and how.

### Documentation Changes

- [ ] Check, test, and update the documentation in [doc/](../doc/). Build documentation (`cd doc/manual/; make html`) to ensure no errors.

### Git Logistics

- [x] Write short & meaningful commit messages. Review each commit for messages (`git show`). Format:
  ```
  topic: description. < 50 characters total.
  
  Longer description. < 70 characters per line
  ```

### Licensing

See [copyright & licensing for more info](https://github.com/m-labs/artiq/blob/master/CONTRIBUTING.rst#copyright-and-sign-off).
ARTIQ files that do not contain a license header are copyrighted by M-Labs Limited and are licensed under LGPLv3+.

### CODE REVIEW

File: artiq/frontend/artiq_ddb_template.py [hunk 1]
```diff
@@ -550,7 +550,7 @@ def process(output, master_description, satellites):
     for peripheral in master_description["peripherals"]:
         n_channels = pm.process(rtio_offset, peripheral)
         rtio_offset += n_channels
-    kasli_with_leds = (master_description["target"] == "kasli" and master_description["hw_rev"] in ("v1.0", "v1.1"))
+    kasli_with_leds = (master_description["target"] == "kasli" and master_description["hw_rev"] in ("v1.0", "v1.1", "v2.0"))
```
Comment (sbourdeauducq) [inline artiq/frontend/artiq_ddb_template.py:553]: I.e. all hardware versions. Remove the check.

File: artiq/frontend/artiq_ddb_template.py [hunk 2]
```diff
@@ -550,7 +550,7 @@ def process(output, master_description, satellites):
     for peripheral in master_description["peripherals"]:
         n_channels = pm.process(rtio_offset, peripheral)
         rtio_offset += n_channels
-    kasli_with_leds = (master_description["target"] == "kasli" and master_description["hw_rev"] in ("v1.0", "v1.1", "v2.0"))
+    kasli_with_leds = (master_description["target"] == "kasli")
```
Comment (sbourdeauducq) [inline artiq/frontend/artiq_ddb_template.py:553]: More code can be removed.

Comment (SidaChen1999) [reply artiq/frontend/artiq_ddb_template.py:553]: You mean that this whole line could be removed?

Comment (sbourdeauducq) [reply artiq/frontend/artiq_ddb_template.py:553]: More than that

**Pull request merged**